tests/Doctrine/Tests/DBAL/Schema/ComparatorTest.php

```diff
@@ -146,6 +146,7 @@ public function testCompareMissingField(): void
         );
         $expected->fromSchema                        = $schema1;
         $expected->changedTables['bugdb']->fromTable = $schema1->getTable('bugdb');
+        $expected->changedTables['bugdb']->toTable   = $schema2->getTable('bugdb');
```

Comment:
The need to pass the `toTable` is a BC break. Why is it necessary in this case?

Reply:
`toTable` is newly defined, what does it break?

Reply:
Why do you need to set it in an existing test?

Reply:
because we are diffing whole object, I would say added property instead of breaking change

Reply:
The need to modify existing tests to keep them passing after a change identifies a BC break.

Reply:
because we are diffing a whole object, which is something not expected to be tested outside DBAL

this change does not break anything in user application except the resulting column order is now like the table was created freshly

if added optional property is considered as BC break, then ok, it is breaking